content/en/docs/ops/common-problems/security-issues/index.md

```diff
@@ -521,33 +521,33 @@ second ensure that [keys and certificates are being delivered](#keys-and-certifi
 If everything appears to be working so far, the next step is to verify that the right [authentication policy](/docs/tasks/security/authentication/authn-policy/)
 is applied and the right destination rules are in place.
 
-## Citadel is not behaving properly {#repairing-citadel}
+## Istiod is not behaving properly {#repairing-istiod}
```

Comment:
Just saying `Istiod is not behaving properly` is too general for this issue.
Maybe something like: `Istiod certificate management is not behaving properly`?
cc @istio/wg-security-maintainers

Reply:
Sounds good.